lib/Doctrine/ORM/Tools/Console/Command/EnsureProductionSettingsCommand.php

```diff
@@ -62,7 +62,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         try {
             $em->getConfiguration()->ensureProductionSettings();
 
-            if ($input->getOption('complete') !== null) {
+            if (!$input->getOption('complete')) {
```

Comment:
current symfony returns `false` not sure if `null` was returned in older version so I did switch to using `!` instead for checking for false or null.

Reply:
There is a default value that can be set in `addOption`. Without an explicit value it is `null`.

https://github.com/symfony/console/blob/5.x/Command/Command.php#L400

It seems to be the same in 4.0 and 3.0.

Reply:
Seems like its not possible to set a default value for InputOption::VALUE_NONE: https://github.com/symfony/console/blob/356c70659d6b48595f9f275cd7c4de0d5544c49c/Input/InputOption.php#L168

Reply:
So I could go with:

```
$input->getOption('complete') === true
```

```
$input->getOption('complete') !== false
```

or the current version.

The value of the input is set here to `true`:
https://github.com/symfony/console/blob/356c70659d6b48595f9f275cd7c4de0d5544c49c/Input/ArgvInput.php#L248

And default value to `false`:

https://github.com/symfony/console/blob/356c70659d6b48595f9f275cd7c4de0d5544c49c/Input/InputOption.php#L182

Reply:
If it's a boolean then your current syntax should fit. You currently let it step into the if-block when the value is `false`. It also looks like there are no tests yet that covers this class. Can you please create one that covers your changes?

Reply:
@SenseException created the missing test class hope it matches how tests are created in this repository.